### PR TITLE
Use a tarball for bzip2

### PR DESCRIPTION
**Pull request by devnoname120**

### CODE REVIEW

File: bzip2/VITABUILD [hunk 1]
```diff
@@ -1,17 +1,18 @@
 pkgname=bzip2
-pkgver=9999
+pkgver=1.0.6
 pkgrel=1
-url="https://github.com/devnoname120/bzip2-vita"
-source=("git://github.com/devnoname120/bzip2-vita.git")
-sha256sums=('SKIP')
+url="http://bzip2.org"
+source=("http://bzip.org/${pkgver}/${pkgname}-${pkgver}.tar.gz" "bzip2.patch")
+sha256sums=('a2848f34fcd5d6cf47def00461fcb528a0484d8edef8208d6d2e2909dc61d9cd' 'SKIP')
```
Comment (xyzz) [inline bzip2/VITABUILD]: `vita-makepkg -g`

File: bzip2/bzip2.patch [hunk 2]
```diff
@@ -0,0 +1,68 @@
+From 1e49af0b2dd7b94f42c2d17eed4d9be832e3cd93 Mon Sep 17 00:00:00 2001
+From: devnoname120 <devnoname120+gh@gmail.com>
+Date: Fri, 26 May 2017 16:57:24 +0200
+Subject: [PATCH 1/2] Vita changes
```
Comment (xyzz) [inline bzip2/bzip2.patch]: can you simplify it into a single change?

File: bzip2/bzip2.patch [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
++AR=arm-vita-eabi-ar
++RANLIB=arm-vita-eabi-ranlib
+ LDFLAGS=
+ 
+ BIGFILES=-D_FILE_OFFSET_BITS=64
+@@ -69,6 +69,10 @@ test: bzip2
+ 	cmp sample3.tst sample3.ref
+ 	@cat words3
+ 
++installlib: libbz2.a
```
Comment (xyzz) [inline bzip2/bzip2.patch]: I suggest removing this part and instead adding these copy commands to the `package()` function.

File: bzip2/VITABUILD [hunk 4]
```diff
@@ -1,17 +1,18 @@
 pkgname=bzip2
-pkgver=9999
+pkgver=1.0.6
 pkgrel=1
-url="https://github.com/devnoname120/bzip2-vita"
-source=("git://github.com/devnoname120/bzip2-vita.git")
-sha256sums=('SKIP')
+url="http://bzip2.org"
```
Comment (xyzz) [inline bzip2/VITABUILD]: the url's wrong

File: bzip2/bzip2.patch [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+--- a/Makefile
++++ b/Makefile
+@@ -15,9 +15,9 @@
+ SHELL=/bin/sh
+ 
+ # To assist in cross-compiling
+-CC=gcc
+-AR=ar
+-RANLIB=ranlib
++CC=arm-vita-eabi-gcc
```
Comment (xyzz) [inline bzip2/bzip2.patch]: instead of this use `make CC=arm-vita-eabi-gcc` etc in the `build()` function

**Comment by xyzz** [comment]
The build isn't actually using arm-vita-eabi-gcc, it's using normal gcc. Try moving `CC=...` to make command line arguments instead of being env vars, like I suggested.

**Comment by devnoname120** [comment]
Done.

**Pull request merged**